src/include/index/index_group.h

```diff
@@ -140,29 +135,12 @@ class base_index_group {
       valid_array_keys_.insert(array_key);
       valid_array_names_.insert(array_name);
       array_key_to_array_name_[array_key] = array_name;
+      array_name_to_uri_[array_name] =
```

Comment:
Why do we need this? We are supposed to get the values from the group after this method is executed

Reply:
Yep I have not yet updated the PR with that change, this is still the original approach

Reply:
@NikolaosPapailiou okay updated now if you want to review!